sklearn/tests/test_multiclass.py

```diff
@@ -64,6 +64,30 @@ def test_ovr_fit_predict():
     assert_greater(np.mean(iris.target == pred), 0.65)
 
 
+def test_ovr_partial_fit_predict():
+    n_iter = 5
+    # make sure partial_fit and regular fit yield the same results
+    ovr1 = OneVsRestClassifier(MultinomialNB())
+    for _ in range(n_iter):
```

Comment:
I find it odd to iterate over the data multiple times with NB. Maybe use ``gen_slices`` and iterate over parts?
Currently it is not testing that anything is stored from previous calls to ``partial_fit``